vortex-layout/Cargo.toml

```diff
@@ -19,8 +19,9 @@ async-once-cell = { workspace = true }
 async-trait = { workspace = true }
 bytes = { workspace = true }
 flatbuffers = { workspace = true }
-futures = { workspace = true }
+futures = { workspace = true, features = ["alloc"] }
```

Comment:
is this feature needed?

Reply:
Yeah, it's needed for futures::channel

Reply:
nvm, its because we disable default features at the top.